UI Session/UI Session 5/css/style.css

```diff
@@ -0,0 +1,84 @@
+header {
+    padding: 20px 0;
+}
+
+header h1, .card h1 {
+    font-size: 2rem;
+    font-family: 'Trade Winds', cursive;
+}
+
+.archivesSection a, .elsewhereSection a {
+    display: block;
+}
+
+main p, main li, .card p {
+    font-family: 'Courgette', cursive;
+}
+
+html {
+    scroll-behavior: smooth;
+}
+
+.card h1 {
+    font-size: 2rem;
+}
+
+.heroSection h1 {
+    font-size: 3rem;
+    font-family: 'Cinzel', serif;
+}
+
+.heroSection p {
+    font-family: 'Cormorant Garamond', serif;
+}
+
+.firehoseSection h1 {
+    font-size: 2.2rem;
+}
+
+.firehoseSection h2 {
```

Comment:
I think class name should be fire-hose-section as it is more readable than firehoseSections.